### ISSUE TITLE
RadioButton's MouseOver/Pressed/Disabled Foreground is always the theme's foreground color

### ISSUE THREAD

**Issue by shinywong** [MahApps/MahApps.Metro#3833]
## Describe the bug

In MahApps.Metro v2.0.0, RadioButton's MouseOver/Pressed/Disabled Foreground is always the theme's foreground color. That is, set RadioButton's Foreground won't change its MouseOver/Pressed/Disabled Foreground. Unlike CheckBox, RadioButton doesn't have properties like "CheckBoxHelper.ForegroundCheckedMouseOver", so I don't know how to change these colors.

## Environment

```
MahApps.Metro version: v2.0.0
Windows build number: Win10 2004 [Version 10.0.19041.264]
Visual Studio: 2019 16.6.1
Target Framework: .NET Framework v4.8
```

**Comment by timunie** [comment]
Hi @shinywong 
at the moment you have to overrdide the `Brushes` on your own:
here are the Brushes: https://github.com/MahApps/MahApps.Metro/blob/2e3954549e185b57e0ef9a14878337db3295f360/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml#L446-L481

**Example**: 
```xaml
<!-- Place this in your Resources, e.g. in App.xaml to override globally -->
<SolidColorBrush x:Key="MahApps.Brushes.RadioButton.ForegroundPointerOver" Color="{DynamicResource MahApps.Colors.Accent}" />
```

Happy coding
Tim

**Comment by shinywong** [comment]
@timunie 
Thank you for the quick reply.
It works.
And I want to know, it is possible to use different colors for Checked/Unchecked?

**Comment by webprofusion-chrisc** [comment]
RadioButton obviously isn't checkbox but is this related to the issue I'm seeing where dark theme checkboxes don't show the checked mark unless you hover? Shows up on Server 2012 R2 desktop. The checkboxes in this image are checked, but they don't look it until the theme is set back to light.

![image](https://user-images.githubusercontent.com/2445502/83939210-c0265400-a80d-11ea-8983-91f030634d8b.png)

![image](https://user-images.githubusercontent.com/2445502/83939245-109db180-a80e-11ea-9311-26ed90f4dd31.png)

**Comment by webprofusion-chrisc** [comment]
Ah, never mind I think my issue is probably fixed in 2.0.1 https://github.com/MahApps/MahApps.Metro/issues/3827
